spark/src/main/java/org/apache/iceberg/actions/BaseSparkAction.java

```diff
@@ -86,6 +125,33 @@
     return manifestDF.union(otherMetadataFileDF).union(manifestListDF);
   }
 
+  protected static Dataset<Row> loadMetadataTable(SparkSession spark, String tableName, String tableLocation,
```

Comment:
I didn't want to make these static, but because two of the actions no longer extend BaseSparkActions they cannot access these methods anymore. I think in the future we should try our best to keep from having Actions implementing  "BaseAction" if we can help it so that the implementations of frame work specific code can use their shared methods.